generator/internal/descriptor_utils.cc

```diff
@@ -836,6 +837,12 @@ VarsDictionary CreateServiceVars(
   vars["connection_header_path"] =
       absl::StrCat(vars["product_path"],
                    ServiceNameToFilePath(descriptor.name()), "_connection.h");
+  vars["connection_rest_cc_path"] = absl::StrCat(
```

Comment:
can we have a unit test, such as:

https://github.com/googleapis/google-cloud-cpp/blob/8680bd69668a9e3b407c56cb103b79c1fa7f5157/generator/internal/descriptor_utils_test.cc#L239-L244

Reply:
Added.